edx_proctoring/backends/rest.py

```diff
@@ -200,6 +201,20 @@ def get_instructor_url(self, course_id, user, exam_id=None, attempt_id=None):
         log.debug('Created instructor url for %r %r %r', course_id, exam_id, attempt_id)
         return url
 
+    def _get_language_headers(self):
+        """
+        Makes the Accept-Language header based on the currently active language
+        """
+        from django.conf import settings
+        from django.utils.translation import get_language
```

Comment:
Importing django here is because we want to wait to import them?  Wouldn't it work to have them at the top?

Reply:
Yes, developers of the subclass may want to import the class, without having to have django installed and configured. Perhaps this function belongs somewhere else?

Reply:
I thought about putting the function elsewhere, but then I'd have to pass in the language to get_attempt, which would require changing the signatures of the other backends. It just seemed a little messier that way.

Reply:
OK, put a comment here explaining the "why".